kentikapi/models/cloud_export.go

```diff
@@ -0,0 +1,243 @@
+package models
+
+// NewAWSCloudExport creates a new AWS CloudExport with all required fields set.
+func NewAWSCloudExport(obj CloudExportAWSRequiredFields) *CloudExport {
+	return &CloudExport{
+		Name:          obj.Name,
+		PlanID:        obj.PlanID,
+		CloudProvider: obj.CloudProvider,
+		AWSProperties: &AWSProperties{
+			Bucket: obj.AWSProperties.Bucket,
+		},
+	}
+}
+
+// NewAzureCloudExport creates a new AZURE CloudExport with all required fields set.
+func NewAzureCloudExport(obj CloudExportAzureRequiredFields) *CloudExport {
+	return &CloudExport{
+		Name:          obj.Name,
+		PlanID:        obj.PlanID,
+		CloudProvider: obj.CloudProvider,
```

Comment:
CloudProvider should be fixed to "azure"

Reply:
Done